config/initializers/devise.rb

```diff
@@ -224,7 +224,7 @@
   # config.navigational_formats = ['*/*', :html]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
because AA is using the `GET` for the `SignOut` link  from header.
But I prefer DELETE also